docs/src/components/action/MuiLogoMenu.tsx

```diff
@@ -0,0 +1,109 @@
+
+interface MuiLogoMenuProps {
+  marginLeft?: boolean;
+  smallerMargin?: boolean;
+}
+
+export default function MuiLogoMenu({ smallerMargin, marginLeft }: MuiLogoMenuProps) {
+  const [contextMenu, setContextMenu] = React.useState<{
+    mouseX: number;
+    mouseY: number;
+  } | null>(null);
+
+  const handleContextMenu = (event: React.MouseEvent) => {
+    event.preventDefault();
+    setContextMenu(
+      contextMenu === null
+        ? {
+            mouseX: event.clientX + 8,
+            mouseY: event.clientY - 8,
+          }
+        : null,
+    );
+  };
+
+  const handleClose = () => {
+    setContextMenu(null);
+  };
+
+  const [copied, setCopied] = React.useState(false);
+  const handleCopy = (svgSnippet: string) => {
+    setCopied(true);
+    copy(svgSnippet).then(() => {
+      setTimeout(() => setCopied(false), 500000);
+      handleClose();
+    });
+  };
+
+  return (
+    <React.Fragment>
+      <Box
+        component={Link}
+        href="/"
+        aria-label="Go to homepage"
+        onContextMenu={handleContextMenu}
+        sx={{ mr: smallerMargin ? 1 : 1.5, ml: marginLeft ? 1.5 : undefined, cursor: 'default' }}
+      >
+        <SvgMuiLogomark height={30} width={30} />
+      </Box>
+      <Menu
+        open={contextMenu !== null}
+        onClose={handleClose}
+        anchorReference="anchorPosition"
+        anchorPosition={
+          contextMenu !== null ? { top: contextMenu.mouseY, left: contextMenu.mouseX } : undefined
+        }
+        sx={(theme) => ({
+          '& .MuiMenuItem-root': {
+            '& * path, .MuiSvgIcon-root': {
+              fill: (theme.vars || theme).palette.text.tertiary,
+              color: (theme.vars || theme).palette.text.tertiary,
+            },
+            '&:hover, &:focus': {
+              '& * path, .MuiSvgIcon-root': {
+                fill: (theme.vars || theme).palette.text.primary,
+                color: (theme.vars || theme).palette.text.primary,
+              },
+            },
+          },
+        })}
+      >
+        <MenuItem onClick={() => handleCopy(logoSvg)}>
+          <SvgMuiLogomark height={16} width={18} sx={{ mr: 1 }} />
+          Copy logo as SVG
+        </MenuItem>
+        <MenuItem onClick={() => handleCopy(logoWordmarkSvg)}>
+          <TextFieldsRoundedIcon sx={{ fontSize: '18px', mr: 1 }} />
+          Copy wordmark as SVG
+        </MenuItem>
+      </Menu>
```

Comment:
I could clearly see a "Visit brand guideline" menu item that points to the right Notion page.

Reply:
Yup, I know. I'm just holding off for a bit as this is still, technically speaking, the organization logo and not the "Material UI" logo. I think it's better to hold off until we have the organization vs. flagship product brand sorted out.

Reply:
I would imagine the branding assets will all be on one page: https://www.notion.so/mui-org/Branding-book-42011f0b6ad44e7ba4e507c00b2da13a?pvs=4, covering Material UI, Pigment CSS, etc so I don't see timeline considerations.

Reply:
If I'm visiting the Material UI docs, I'd expect to have its brand assets on its docs space instead of somewhere else (where I'd see brand assets from other products). The timeline impact that I see is:

- You're in mui.com, which is technically the _org_ website. You right-click and see the "View brand guidelines" menu item that redirects you to that page.
- On that page, we'd display the MUI logo with the "MUI" wordmark instead of "Material UI" because we haven't fully/firmly decided on a new company name to unlock the logo back to Material UI.
- Given that the menu item is also accessible via the Material UI docs space, if you're redirected to a page that tells you that logo means MUI, that 200% cement that MUI = Material UI. I'm totally up for doing this, but I'd prefer doing it once it doesn't conflict with the organization's brand (whatever that is).

Reply:
Yeah OK, maybe. I guess I feel like Notion is good enough for this. It's simpler to execute, faster to keep up to date, and it match the need for this, that feels rare.